packages/contracts/tasks/take-dump.ts

```diff
@@ -128,6 +129,40 @@ task('take-dump').setAction(async (args, hre) => {
     commit = '0000000000000000000000000000000000000000'
   }
 
+  // Insert prefunded accounts into the state dump when running on the local
+  // network
+  if (hre.network.name === 'local') {
+    const accounts = [
```

Comment:
Shouldn't https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/deploy/016-fund-accounts.ts be doing this?

Reply:
Also you can get these addresses like so: https://github.com/ethereum-optimism/optimism/blob/23ed59b2dacd32227fb6a7382b53e1a8f30e6a73/packages/contracts/deploy/016-fund-accounts.ts#L27

Reply:
I can double check